hubmap/frontend/src/components/home/Search.js

```diff
@@ -158,7 +158,7 @@ class SearchBox extends React.Component {
                                                     100% {fill: ${entry[1]}; opacity: 1;}`);
                                                     tissueElement.style.removeProperty("animation");
                                                     tissueElement.addEventListener("animationend", animationEnd);
-                                                    tissueElement.style.setProperty("animation", tissue + 'tissueAnimation' + " 10s linear");
+                                                    tissueElement.style.setProperty("animation", tissue + "tissueAnimation 10s linear");
```

Comment:
These are all good, but it will make things easier if each PR stays on topic:
- In the git history, it will be easier to understand if this comes from a PR just about formatting.
- And in reviewing a PR that's already complicated, it's good to keep it as focused as possible.

Reply:
Honestly, this errors only showed up in this branch, am sure, there is some branch that would have fixed this eslint errors but is not merged into develop branch. So what might have been fixed in one branch might be different, not sure, I see same errors over and over again. I see travis build fails without this fix every time, as we could expect.

Reply:
develop branch commit #0857f9c1f987a6e9dd995619d8f46282e664de46 auto-fixes this.